packages/@lwc/engine/src/framework/mutation-tracker.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { valueMutated, valueObserved } from '../libs/mutation-tracker';
+import { VM } from './vm';
+
+export function componentValueMutated(vm: VM, key: PropertyKey) {
+    valueMutated(vm.component, key);
+}
+
+export function componentValueObserved(vm: VM, key: PropertyKey) {
+    valueObserved(vm.component, key);
+}
+
+export * from '../libs/mutation-tracker';
```

Comment:
valueMutated and valueObserved are not used by the engine directly anymore... this export should be removed.

Reply:
yes, they are used in `membrane.ts`: https://github.com/salesforce/lwc/pull/1459/files#diff-c34c5e3e3f4645ec4cf4cf59652dedd6R8